File: 프로그래머스/0/120822. 문자열 뒤집기/문자열 뒤집기.java
import java.lang.StringBuilder;

class Solution {
    public String solution(String my_string) {
        // String answer = "";
        // char[] array = my_string.toCharArray();
        // for(int i = array.length-1; i>=0; i--){
        //     answer += array[i];
        // }
        // return answer;
        // 다른 코드
        StringBuilder sb = new StringBuilder(my_string);
        return sb.reverse().toString();
    }
}

File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.py
def solution(my_string, s, e):
    arr = list(my_string)
    tmp = list(my_string)
    tmp = tmp[s:e+1] # ['2', '1', 'S', 'r', 'e', 'm', 'm']
    print(tmp)
    tmp = tmp[::-1]
    print(tmp) # ['m', 'm', 'e', 'r', 'S', '1', '2']
    arr[s:e+1] = tmp
    print(tmp)
    arr = ''.join(arr)
    print(arr)
    return arr